lib/rucio/common/.vim_runtime/my_configs.vim

```diff
@@ -0,0 +1 @@
+set mouse-=a
```

Comment:
I think this file shouldn't make it into the repo

Reply:
No, not yet until the the propper levels for testing and server mode are set. Pls check: #3912

Reply:
No, that vim file shouldn't be in the repo at all 😄